pkg/scheduler/framework/runtime/framework.go

```diff
@@ -724,7 +725,7 @@ func (f *frameworkImpl) RunPreFilterPlugins(ctx context.Context, state *framewor
 				returnStatus = s
 				continue
 			}
-			return nil, framework.AsStatus(fmt.Errorf("running PreFilter plugin %q: %w", pl.Name(), s.AsError())).WithPlugin(pl.Name())
+			return nil, framework.AsStatus(fmt.Errorf("running PreFilter plugin %q: %w", pl.Name(), s.AsError())).WithPlugin(pl.Name()), unscheduledPlugins
```

Comment:
If some plugins rejects, we should make `unscheduledPlugins` empty so that they won't be registered in `diagnosis.UnschedulablePlugins`.
I know the current implementation already does that, but let's return nil explicitly.

```suggestion
			return nil, framework.AsStatus(fmt.Errorf("running PreFilter plugin %q: %w", pl.Name(), s.AsError())).WithPlugin(pl.Name()), nil
```